### PR TITLE
Fix various linter errors

### PR DESCRIPTION
**Pull request by nmittler**

### CODE REVIEW

File: tests/e2e/tests/pilot/cloudfoundry/copilot_test.go [hunk 1]
```diff
@@ -337,7 +337,7 @@ func newTestState(mockCopilotServerAddress string, servicePort int) *testState {
 	creds := testhelpers.GenerateMTLS()
 	clientTLSFiles := creds.CreateClientTLSFiles()
 	return &testState{
-		creds: creds,
+		creds:                 creds,
```
Comment (fisherxu) [inline tests/e2e/tests/pilot/cloudfoundry/copilot_test.go]: this seems to be the go fmt err here

**Comment by nmittler** [comment]
@hklai running `make lint` locally works fine.  Any ideas?

**Comment by hklai** [comment]
> @hklai running make lint locally works fine. Any ideas?

@nmittler I have seen something like this when circle downloads a different (newer) version of linter.

**Comment by nmittler** [comment]
@hklai who owns the linter check? I'm simply running `make lint` from the command line, which should be dealing with version, etc.

**Comment by GregHanson** [comment]
@nmittler `bin/fmt.sh`uses go imports.  you can run `which go imports` and delete the corresponding binary. Remove `golang.org/x/tools/cmd/goimports` from $GOPATH/src.  And delete $GOPATH/bin and $GOPATH/pkg.  then you can try the `go get golang.org/x/tools/cmd/goimports` command again

**Comment by hklai** [comment]
@nmittler I think the earlier linter problem had been fixed. Perhaps you can rebase and try this again.

One thing I don't quite understand is that when I run make lint, I ran into the same linter failures that you are fixing here. I wonder how the linter jobs are passing in circle....

**Comment by nmittler** [comment]
@hklai that's my concern as well ... I've tried from 2 separate machines and had similar results. It seems that something is wrong/out-dated with the Circle configuration.

**Comment by sdake** [comment]
for extra data - linter is failing for me locally as well, and it appears to be faling in the gate.

**Comment by nmittler** [comment]
Closing this, since the linter issues seem to have been addressed.

**Pull request closed without merging**